cypress/page/shipping-step.page.ts

```diff
@@ -4,7 +4,9 @@ class ShippingStepPage {
 
     constructor() {
         this.check = "[name=cgv]";
-        this.proceedToCheckoutAdress = ".cart_navigation span";
+        this.proceedToCheckoutAdress = "[type='submit'][name='processCarrier']";
```

Comment:
here you can use the html tag + the title: `button[name='processCarrier']`

Reply:
Ok, thank you! But again, I have the same question, isn´t it better to have two attributes than a tag and the attribute?